src/component/qrcode-scanner.js

```diff
@@ -62,12 +67,19 @@ export default class QRCodeScanner extends React.Component {
     } else {
       return (
         <View style={styles.wrapper}>
-          <BarCodeScanner
-            onBarCodeScanned={
+          <RNCamera
+            captureAudio={false}
+            androidCameraPermissionOptions={{
+              title: 'Permission to use camera',
+              message: 'Allow Lightning to use the camera',
+              buttonPositive: 'OK',
+              buttonNegative: 'Cancel',
```

Comment:
Do we navigate back if the user presses "Cancel"?

Reply:
That just declines the camera permission. No need to navigate back.